<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="course_field">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>60</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>course ID, course name, course responsible</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="coursedate_field">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>year, month, day</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="coursetime_field">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>hour, minute</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="courselocation_field">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>location ID,  address</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="showresult">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="course_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Course:</string>
    </property>
   </widget>
   <widget class="QLabel" name="coursedate_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Course Date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="coursetime_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Course Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="courseloacation_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Course Location:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitbtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitbtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>490</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearbtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
